pkg/controller/nebulascheduledbackup/nebula_scheduled_backup_manager.go

```diff
@@ -0,0 +1,250 @@
+/*
+Copyright 2023 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nebulascheduledbackup
+
+import (
+	"fmt"
+
+	"github.com/robfig/cron"
+	"k8s.io/klog/v2"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/vesoft-inc/nebula-operator/apis/apps/v1alpha1"
+	"github.com/vesoft-inc/nebula-operator/pkg/kube"
+	condutil "github.com/vesoft-inc/nebula-operator/pkg/util/condition"
+	utilerrors "github.com/vesoft-inc/nebula-operator/pkg/util/errors"
+)
+
+const (
+	EnvS3AccessKeyName = "S3_ACCESS_KEY"
+	EnvS3SecretKeyName = "S3_SECRET_KEY"
```

Comment:
Why not use the officially recommended environment variable names?
ref: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

Reply:
Fixed